akka-http/client/src/main/scala/endpoints/akkahttp/client/StatusCodes.scala

```diff
@@ -13,11 +13,15 @@ trait StatusCodes extends algebra.StatusCodes {
   type StatusCode = AkkaStatusCode
 
   def OK = AkkaStatusCodes.OK
+  def Created = AkkaStatusCodes.Created
```

Comment:
```suggestion
  val Created = AkkaStatusCodes.Created
```

Reply:
There is a subtlety here. If we make it a `val` we become subject to initialization errors, in case the val is used before it is effectively initialized. So, it should be a `lazy val`. However, `lazy val`s have some overhead when they are read. Since, in this case, we are just forwarding to something else, I’ve kept a `def`: there is nothing expensive to compute, so in case the JIT does not inline the `def`, we don’t pay a huge price.